Sources/Components/SaveSearchView/SearchListEmptyView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+// MARK: - SearchListEmptyViewDelegate
+
+public protocol SearchListEmptyViewDelegate: AnyObject {
+    func searchListEmptyViewDidSelectActionButton(_ searchListEmptyView: SearchListEmptyView, forState state: SearchListEmptyView.SearchListEmptyViewState)
+}
+
+@objc public class SearchListEmptyView: UIView {
+
+    // MARK: - Public properties
+
+    @objc public enum SearchListEmptyViewState: Int {
```

Comment:
Does this need to be exposed to objc? It doesn't seem to be used in the PR for the app.

To shorten the name, since it's already a nested enum, how about naming it `ViewState`?